addons/html_editor/static/tests/unbreakable/delete.test.js

```diff
@@ -263,29 +263,22 @@ describe("backward", () => {
             });
         });
 
-        describe("triple click", () => {
-            test("should delete last character of paragraph, ignoring the selected paragraph break leading to an unbreakable (1)", async () => {
-                await testEditor({
-                    contentBefore: `<p>ab[c</p><p class="oe_unbreakable">]def</p>`,
-                    // This type of selection (typically done with a triple
-                    // click) is "corrected" before remove so triple clicking
-                    // doesn't remove a paragraph break.
-                    stepFunction: deleteBackward,
-                    contentAfter: `<p>ab[]</p><p class="oe_unbreakable">def</p>`,
-                });
+        test("should delete last character of paragraph and the empty unbreakable (1)", async () => {
```

Comment:
There's no empty unbreakable here. I guess it comes from the test below...
I suggest "should delete last character of paragraph but not merge it with unbreakable"